### PR TITLE
Adds voltine as a fluff chem of majestic fluffery (WIP)

### PR DESCRIPTION
**Pull request by Ar3nn**
*prepares anti-snow bunker*

:cl:
rscadd: Adds new fluff chem for IPC, which is a substitute for nicotine in their cigarettes
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
:-1: 

The whole point of synthetic is that they have unique upsides and downsides. One of their upsides is immunity to the vast majority of chems.

That said, this comes at a cost; they have very very few beneficial chems available to them, and the ones they do have are generally tedious to make.

This is just pushing a strength even further when when IPC's chem processing is meant to be more of a uniqueness.

IPCs are more than capable of smoking already for RP purposes.

**Comment by Ar3nn** [comment]
I'd be open to alternatives, but this particular thing has always irked me as a "why can't everyone have this nice thing equally" since its so prevalently available around the station.

**Comment by Fox-McCloud** [comment]
>"why can't everyone have this nice thing equally"

Because IPCs are meant to be different, and they already have unique upsides by not processing the vast majority of nasty/horrifically lethal chems that everyone else has to put up with.

This is just pushing more power into IPCs for no reason at all.

**Comment by Ar3nn** [comment]
Your entire argument is that not processing chems is a buff, but I see it as a rather neutral effect to begin with, since there's also positive chems we do not benefit from. I'm not extreme enough to shade it a nerf, but it's definitely not all ups for IPC.

**Comment by KasparoVy** [comment]
Not processing chems is such a wildly huge buff it's not even funny. Well, it might be in certain situations. Yeah you can't bathe in styptic and silver sulphadiazine and be good after getting shot up by an LMG and burnt to a crisp by a flamethrower, but John McMeme isn't going to waste you with his syringe/smoke/whatever full of kek mix in two seconds from full health unless he's got a backup EMP.

FlattestGuitar already said my sentiments on this

**Comment by Ar3nn** [comment]
Fox already laughed at my reasoning for thinking it should probably be a thing, so fluffchems it is. Everyone's extremely touchy about giving IPC anything, but they don't wanna nerf them either.

**Comment by IcyV** [comment]
"The whole point of synthetic is that they have unique upsides and downsides. One of their upsides is immunity to the vast majority of chems."

IPCs were created with this in mind. They were supposed to be robotic things that were super brittle but immune to most organic issues. Now? They have their own special chems for many things that mimic some of the major organic ones, death and healing is practically a non-issue with how fast they can heal themselves on top of the sheer amount of damage it takes to kill them.

IPCs get great bonuses in their immunity to so many basic things. They shouldn't get literally everything that organics get without the downsides.

**Comment by Spacemanspark** [comment]
:-1: Fox describes what my feelings are on this perfectly.

**Comment by Ar3nn** [comment]
I love how I changed the name to fluff chem of flufftasticy and put fluffernuts in the changelog and people are STILL FUCKING BITCHING about something I removed from the code four commits ago.

**Comment by Regen1** [comment]
They're trying to explain to your narrow sighted mind why they're not agreeing with your initial change.

You're just so stuck up with being a victim of the "IPC hate brigate" to realize it.

**Comment by TheDZD** [comment]
I honestly don't like the pattern of making more and more special robutt drugs for IPCs, whether purely for fluff reasons or not. They're robots, the big thing about them is that they don't process most medical reagents/drugs, if you want to smoke yourself to death go play something that processes chems normally.

**Comment by Ar3nn** [comment]
@Regen1 Well, I mean, it's sort of valid that there kind of is a hate brigade. Any change to IPC is looked at with trepidation, because IPC is so weird and different, nobody really knows what to do to make it more reasonable, and nobody wants to try, and yet anyone that tries gets slammed with a wall of "ermahgerd snowflake, ermahgerd stop buffing most op race" and this was probably going to be used even less than cigarettes in the first place. I dunno why this mindset exists, but maybe a total refactor of IPC wouldn't be too unjustified, due to the way the coderbase looks towards anything even tangentially related to them.

@TheDZD Yeah, I can see that, fluffchems just to say oh lookie we can process this is probably needless fuckery, so I'm gonna abandon that. I guess I switched just to say "my work didn't go totally to waste."

TL;DR /PR, ya'll can keep your homeostasis.
